conf/local.conf.sample

```diff
@@ -168,15 +168,6 @@ TOOLCHAIN_HOST_TASK:remove = "${TOOLCHAIN_HOST_REMOVE}"
 # example, to add libncurses:
 #TOOLCHAIN_TARGET_TASK_EXTRA += " ncurses-libncurses"
 
-# Uncomment to set a title for the SDK
-# Default: SDK for ${SDK_IDENTIFIER}
-#SDK_TITLE ?= "My Company's SDK for ${SDK_IDENTIFIER}"
-
-# Uncomment to alter the identifier for the SDK. This mechanism is used to
-# isolate the SDK on disk, as well as the installers.
-# Default: ${SDKMACHINE}-${IMAGE_BASENAME}-${MACHINE}-${SDK_VERSION}
-#SDK_IDENTIFIER ?= "${SDKMACHINE}-${IMAGE_BASENAME}-${MACHINE}-${SDK_VERSION}.customized"
```

Comment:
Why'd we have these in the first place?

Reply:
Originally the ADEs weren't installed into isolated locations, the identifier controlled it, so we allowed for customization to control whether it'd replace/upgrade an existing ADE or be isolated. That's since changed. And of course the ade title/sitename and identifier were both user visible in the install/remove software dialog. Now the identifier is used by codebench and is in the install path, but that's about it, and the title/sitename is only visible in the shell archive installer, and doesn't mention company name, so the user is unlikely to want to change something so trivial to their own.